### PR TITLE
fix #5, add missing footer links

### LINKED ISSUE #5: Footer with links
It's a common pattern to have a footer with all the main internal and external links of the site.

### PR DESCRIPTION
**Pull request by DerekNonGeneric**
Notes:
&emsp;https://user-images.githubusercontent.com/17770407/89892605-e87f5580-dba4-11ea-972d-677df535b31e.png
&emsp;https://user-images.githubusercontent.com/17770407/89892894-617ead00-dba5-11ea-974b-486159effa7f.png

Fixes: #5

### CODE REVIEW

File: _layouts/page_footer.html [hunk 1]
```diff
@@ -1,7 +1,77 @@
...
+      </div>
+      <div class="g-col-3">
+        <h3>Proposals</h3> <!-- (list of proposals and their state) -->
+        <ul>
+          <li><a href="https://github.com/tc39/proposals/blob/master/stage-0-proposals.md">Stage 0</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/stage-1-proposals.md">Stage 1</a></li>
+          <li><a href="https://github.com/tc39/proposals#stage-2">Stage 2</a></li>
+          <li><a href="https://github.com/tc39/proposals#stage-3">Stage 3</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/finished-proposals.md">Stage 4 (Finished)</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/inactive-proposals.md">Inactive</a></li>
```
Comment (DerekNonGeneric) [inline _layouts/page_footer.html]: I would like to put the contents of these Markdown files into data structures.

**Comment by DerekNonGeneric** [comment]
We could also take the footer in a different direction. Please give this a thumbs up if y'all think this looks nicer (I do).

Screenshot_2020-07-30 TC39 – Specifying JavaScript (3)

File: _layouts/page_footer.html (hunk 1 above)
Comment (codehag) [reply _layouts/page_footer.html]: what do you mean here? do you mean a rendered view on the website?

**Comment by DerekNonGeneric** [comment]
Please disregard my comment above, but a hint as to whether or not this PR is properly addressing #5 would be nice. :)

/cc @codehag

**Comment by codehag** [comment]
Ah, i thought i reviewed this on the weekend. Probably i didn't merge because it was marked as draft.

It looks great! thanks for all the work!

**Review by codehag — APPROVED** [review]

**Pull request merged**